services/director-v2/tests/mocks/fake_scheduler_data_compose_spec.json

```diff
@@ -97,5 +97,6 @@
   },
   "request_dns": "localhost",
   "request_scheme": "http",
-  "proxy_service_name": "dy-proxy_dd2b8ceb-4408-4bfb-a953-46178836e12d"
+  "proxy_service_name": "dy-proxy_dd2b8ceb-4408-4bfb-a953-46178836e12d",
+  "request_simcore_user_agent": ""
```

Comment:
are you by chance also mocking a compose spec where `request_simcore_user_agent` is filled? Might be cool, might be useless, you decide ;)

Reply:
Having or not having a value is equivalent. An empty string is the same as a string with more that one characters from a post of view of Pydantic, which requires the field to be passed in.
I think this is ok.
There are other unit tests for the model that take care of that.